jujugui/static/gui/src/test/test_bundle_import_notifications.js

```diff
@@ -124,168 +124,6 @@ with this program.  If not, see <http://www.gnu.org/licenses/>.
     });
   });
 
-  describe('bundle helpers watchAll', function() {
```

Comment:
Back in the olden times we used to get deltas on bundle imports - they weren't just individual api calls.